ingest/defaults/segment_resolutions.yaml

```diff
@@ -0,0 +1,27 @@
+PICK_SEGMENT:
+  - strain: TRVL9760
+    accession: KP026181 # matches the metadata for the other segments for this strain
+    segment: S
+PICK_FIELD:
+  - strain: H498913
+    field: date
+    accession: HQ830423 # 2 accessions have the same date of 1988, one has 1990. So we choose majority rule (as an example).
+  # following authors are very similar, pick the most complete looking
+  - strain: LET-2083
```

Comment:
@miparedes, re:

> does this new workflow output a file, or even to the terminal, the strains that are being deduped? 

Yes - it prints to the terminal. Logging can (and will) be improved as this PR matures.

> I ask because I noticed in the builds that are staged that all the Colombia seqs are missing and I can't really tell if that's because they were dropped because theyre duplicates or not.

Those samples weren't included in the build I pushed to staging as they were skipped due to conflicting metadata (one example was in the PR message, but it wasn't obviously from Columbia: "ERROR! 'LET-2083' Disagreement...").

> Either way, even if some are duplicated I'd expect at least one of those seqs to make it through?

There's no automatic conflict resolution (yet), however they will now make it through as the YAML file (where I've attached this comment) resolves the conflicts, but this improvement was pushed after I made the initial (staging) build.

Reply:
(I've reworked this PR and re-uploaded intermediate files to S3 and auspice JSONs to staging, so the Columbia sequences are now there...)